### ISSUE TITLE
Indicates file exists even though it is not present

### ISSUE THREAD

**Issue by Leucocoryne** [Nandaka/PixivUtil2#782]
If you remove a file that you downloaded once from the folder and then download it again using the "alwayscheckfile size = True" option, it will appear as "exist" even if the image does not exist within the folder. Also, it is not downloaded.

Is there any way to fix this without deleting "db.sqlite"?

**Comment by Nandaka** [comment]
config/log file? also `alwayscheckfilesize = True` (no space).

I just checked using option `2`, download image an image, delete it, then redownload=> it will download again.

**Comment by Leucocoryne** [comment]
Attached is "db.sqlite".
Add this file to the new version of PixivUtil2.
config file changed only the "alwayscheckfilesize = True" option.

Download files that you've downloaded in the past as "Download by image_id".

ex) 83894756, 82493437

It appears as "exist" even though the file is not in the folder, and the download is not possible.

Strangely enough, the files that were recently downloaded and removed were normally downloaded again.


================
[db.zip](https://github.com/Nandaka/PixivUtil2/files/5165620/db.zip)
================

**Comment by Nandaka** [comment]
I need the log file, it should be called `pixivutil.log` in the same folder. You can delete the old log and rerun to simulate the error.

Other solution is just delete your db to redownload the image again.

**Comment by Leucocoryne** [comment]
I will attach the log file you mentioned.

I tried using the previously attached DB file.
The display indicates that the file exists.
But only the folder is created and the image is not downloaded again.

I'm afraid...
There seems to be no other way but to delete the DB file.
I don't know why this is happening.

[pixivutil.log](https://github.com/Nandaka/PixivUtil2/files/5175331/pixivutil.log)

**Comment by Nandaka** [comment]
hmm, checked with the same image, I suspect because it is exists in different folder. I notice you are using %artist% as the filename format, so it is possible this is the case. 

Try to search using the image id in your download folder? I will update the message to show the file location for the warning message.

I recomment to use `%member_token%`, as it is less likely to change compared to `%artist%`

**Comment by Nandaka** [comment]
You also can try to clean up the database using DB Managed (d) -> Clean Up Database (c).

**Comment by Leucocoryne** [comment]
Ah... I think I know what's wrong.
Usually when I have an update of Pixivutil2
I moved the DB file by pasting the "db.sqlite" file from the previous folder "pixivutil20200726" to the "pixivutil20200827-beta1" folder.

Now, even if you run the latest exe file in the "pixivutil20200827-beta1" folder to find the image, you do not find duplicate files in the "pixivutil20200827-beta1" folder due to the influence of the previous DB file.
When I checked the file in the previous folder "pixivutil20200726", it was a problem to mark it as existed because it existed.
